Extensions/TeamCity/Src/vss-extension.json

```diff
@@ -201,7 +211,8 @@
           },
           {
             "target": "artifactDetails",
-            "resultTemplate": "{ Name: \"{{version}}\", StreamType: \"zip\", downloadUrl : \"{{endpoint.url}}downloadArtifacts.html?buildTypeId={{definition}}&buildId={{version}}\" }"
```

Comment:
We are not really using the "downloadUrl" or the "StreamType" later on. Moreover the template doesn't contain the content of the file. So we changed it. Let me know if any changes need to be made.